### ISSUE TITLE
Excel file view in files pane does not work - MacOS Big Sur

### ISSUE THREAD

**Issue by jcordovilla** [rstudio/rstudio#8604]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.1093
    OS Version      : MacOS Big Sur 11.0.1
    R Version       : 4.0.2

### Steps to reproduce the problem
I am running an R project to practice with the Tidyverse packages.
I have a raw data folder with an excel (.xlsx) file that I want to inspect before importing as dataset.
In the files pane I click on the excel file and the two options appear: "View File" and "Import Dataset". The latter works fine, but the former has no effect; no reaction to clicking on it.
Thanks.
<img width="1280" alt="Captura de pantalla 2020-12-12 a las 0 26 43" src="https://user-images.githubusercontent.com/35451373/101964546-0ff93400-3c12-11eb-80b6-438857c11ceb.png">

### Describe the problem in detail
Clicking on "View File" does nothing.

### Describe the behavior you expected
I expected the file to open in Microsoft Excel. The xlsx extension is associated to Excel in my system.

- [ x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kevinushey** [comment]
Do you also see this in the latest [preview release](https://www.rstudio.com/products/rstudio/download/preview/)?

**Comment by jcordovilla** [comment]
Hi @kevinushey . I just installed the preview release and, oddly enough, now the excel files do not even show up in the files pane, although it is actually there. See screenshots of RStudio and Finder windows. It seems that RStudio is somehow filtering the filetypes.
Any suggestions? Thanks.
<img width="1729" alt="imagen" src="https://user-images.githubusercontent.com/35451373/102125648-4e793380-3e4a-11eb-82ed-2a055a169539.png">
<img width="1304" alt="imagen" src="https://user-images.githubusercontent.com/35451373/102125774-723c7980-3e4a-11eb-930e-3234aad41bff.png">

**Comment by kevinushey** [comment]
That is bizarre! I haven't seen that before. Does the file appear if you explicitly refresh the Files pane? Is this true for all `.xlsx` files, regardless of their location on the filesystem?

**Comment by jcordovilla** [comment]
Aha! I found the problem: I had tried to refresh the Files pane without any joy.
However, I then followed you suggestion to try with other excel files elswhere and noticed that certain folders in my R directory -which is synced via iCloud- were not locally stored in full. I.e. some sub-folders and files show in MacOS's Finder app but are not actually downloaded onto the local drive to optimise storage. These files are marked by the little cloud with down arrow icon (see pic).
Once I clicked on the icon and the R folder was fully downloaded the xslx file in question revealed itself!
Funny enough, I checked whether the same issue happens with Google Drive File Stream, which has the same local/cloud storage option. RStudio does see the files in Drive FS even when not stored locally.
Hope the above info is useful.
The "View" option that I had originally reported is now working in all cases, by the way. Many thanks. Jose
<img width="1304" alt="Captura de pantalla 2020-12-15 a las 0 35 42" src="https://user-images.githubusercontent.com/35451373/102148870-8ba2ed00-3e6d-11eb-96a7-b3523655fd66.png">
